packages/ui/src/components/va-file-upload/VaFileUpload.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
Do we even need red crosses/bins for disabled items? I think they only add to confusion and we can remove them safely?